docker-compose.yml

```diff
@@ -44,3 +44,8 @@ services:
     volumes:
       - ./data/nylas_part:/var/lib/inboxapp
     hostname: sync-engine-start
+
+networks:
```

Comment:
Is it necessary? At least it works without specifying **networks** in both local and production environments.
There's another compose file for production environment: **docker-compose-production.yml**. Perhaps, it should added in both compose files

Reply:
It's not necessary(users can join container to the network manually), but it lil bit automates setup of local env. It's needed for getting stitch and nylas work at the same network.